ckan/logic/action/create.py

```diff
@@ -774,18 +775,17 @@ def _group_or_org_create(context: Context,
         'model': model,
         'user': user,
         'ignore_auth': True,  # we are not a member of the group at this point
-        'session': session
+        'session': session,
+        'defer_commit': context.get("defer_commit") or False,
```

Comment:
As discussed at the dev call if we're going to more widely support `defer_commit` across actions let's add it to the `fresh_context` function and use that one instead of the code here